Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -900,7 +948,7 @@
           DefaultPrototype: SaxophoneInstrument
           ExamineName: Woodwind Instrument
     - type: ReverseEngineering # Nyano
-      difficulty: 3
+      difficulty: 2
```

Comment:
why is a bio fabricator harder to research than this?

Reply:
This is lower on thr reserch teirs, bio fabs are teir two on civ reserch. While daws are teir one